modules/icress_module.php

```diff
@@ -39,26 +39,28 @@ function icress_getJadual() {
 }
 
 function icress_getCampus($campus, $faculty) {
-		$form_names = getFormNames();
+		// $form_names = getFormNames();
 
 		$postdata = http_build_query(
 				array(
-						$form_names['search_campus'] => $campus,
-						$form_names['search_faculty'] => $faculty
+						// $form_names['search_campus'] => $campus,
```

Comment:
The change back the dynamic value of these two keys to `search_campus` and `search_course`